pkg/cagent/scripts/pkg-deb-build.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Docker Packaging authors
```

Comment:
Does the check check for year, or is that a wildcard? Otherwise this could be 2025 (year it was added for this file).

Reply:
Yes should be 2025